Library/Homebrew/cask/dsl/caveats.rb

```diff
@@ -58,6 +58,19 @@ def eval_caveats(&block)
         EOS
       end
 
+      caveat :unsigned_accessibility do |access = "Accessibility"|
+        # access: the category in System Preferences -> Security & Privacy -> Privacy the app requires.
+
+        <<~EOS
+          #{@cask} is not signed and requires Accessibility access,
+          so you will need to re-grant Accessibility access every time the app is updated.
+
+          Enable or re-enable it in:
+            System Preferences → Security & Privacy → Privacy -> #{access}
```

Comment:
I considered actually running `open /System/Library/PreferencePanes/Security.prefPane` to open the pane for the user, but that would get run even if you run `brew info` AIUI, so decided against it.

Also didn't document that as having two ways to do the first step is needlessly confusing.